deployment_and_ssl_doc.md

```diff
@@ -239,13 +331,19 @@ When you have done this press the
 followed by  
 `:wq`
 
-Now on the virtual machine run the command  
+On the virtual machine run the command  
 `service nginx reload`
 
-Now exit your sever by typing the command `exit`.
+Exit your server by typing the command `exit`.
 
 ## Add super_admin user to database
 
+**Note:** You do not need to add a `super_admin` for
+[Healthlocker](https://github.com/healthlocker/healthlocker/blob/master/deployment_and_ssl.md).
+However, there are other roles which may need to be updated. Please see the
+Healthlocker deployment guide for more details.
+
+
 On the virtual machine run the commands
```

Comment:
Should the following `super_admin` instructions refer to the HealthLocker document for differences?

Reply:
It does already. Starts at line 342.

Reply:
Sorry - my bad